lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -77,7 +77,7 @@ fields:
     description: The clinical status of the allergy or intolerance.
     terminology:
     - display: AllergyIntolerance Clinical Status Codes
-      system: http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical
+      system: http://terminology.hl7.org/CodeSystem/aller gyintolerance-clinical
```

Comment:
nit: Spelling

Reply:
my bad reverted the change https://github.com/cerner/fhir.cerner.com/pull/325/commits/96dd8b454f4f1f5d3b5ef538fbbafdb623da089d